tds/src/test/java/thredds/server/ncss/dataservice/FeatureDatasetTypeTest.java

```diff
@@ -110,4 +110,13 @@ public void getFDForFeatureCollectionStationDataset() throws IOException{
 		assertNotNull(fd);
 		assertEquals(FeatureType.STATION, fd.getFeatureType());
 	}
+	@Test
+	public void getFDForSimpleGeometryFeatureDataset() throws IOException{
+		MockHttpServletRequest req = new MockHttpServletRequest();
+		MockHttpServletResponse res = new MockHttpServletResponse();
+
+		FeatureDataset fd = TdsRequestedDataset.getSimpleGeometryFeatureDataset(req, res, "/thredds/cdm/src/test/data/dataset/SimpleGeos/hru_soil_moist_3hru_5timestep.nc");
+		assertNotNull(fd);
+		assertEquals(FeatureType.SIMPLE_GEOMETRY, fd.getFeatureType());
```

Comment:
Good assertions! Is it relevant to cast `fd` to `SimpleGeometryFeatureDataset` for purposes of adding more Simple Geometry-specific assertions? If you find that `SimpleGeometryFeatureDataset`'s functionality is adequately tested in other test classes, then there's no need to expand on these assertions, but if you think it would improve robustness, you are welcome to add more assertions.